scenario/time_sync/synchronization.rst

```diff
@@ -1,4 +1,141 @@
 How to do synchronization
 =========================
 
-TBD
+
+We has provided a daemon to tigger camera with external signal for **Frame Synchronization**.
+
+* MTi 670/680 with SyncOut ⇒ Send 1 PPS
+* ROScube X with mraa ⇒ GPIO ⇒ Trigger ISR + Camera
+
+Interfacing
+^^^^^^^^^^^
+
+Refer form `Interfacing an MTi GNSS/INS device with a Velodyne Lidar <https://base.xsens.com/s/article/Interfacing-an-MTi-GNSS-INS-device-with-a-Velodyne-Lidar?language=en_US>`_.
+
+The figure shows how to connect your MTi-6x0 to the Velodyne Interface Box.
+
+Please note the following:
+
+* For testing purposes it is possible to power the MTi-6x0G directly using the 12V supply available in the Interface Box.
+* As mentioned in the MTi 600-series Hardware Integration Manual, the RS232 CTS line of the MTi-6x0G needs to be tied to a logical high (3-25V). Otherwise the MTi will not transmit data over the RS232 interface. 
+
+.. image:: images/interface-box.png
+    :width: 80%
+    :align: center
+
+Building
+^^^^^^^^
+
+Building the daemon from github.
+
+Get the code
+------------
+
+.. code-block:: bash
+
+    cd ~
+    git clone https://github.com/Adlink-ROS/camera_trigger_daemon.git
+    cd camera_trigger_daemon
+
+Installation
```

Comment:
I think you just need to have `sudo apt install neuron-library` here.
For more information, link to our Neuron Library page.

Reply:
OK, I will link our page here.